PokeAlarm/Twitter/TwitterAlarm.py

```diff
@@ -70,8 +73,38 @@ def set_alert(self, settings, default):
 
     # Post Pokemon Status
     def send_alert(self, alert, info):
-        args = {"status": replace(alert['status'], info)}
-        try_sending(log, self.connect, "Twitter", self.__client.statuses.update, args)
+        # Optimize remaining tweet length when removing <gmaps>
+        # There must be spaces around <gmaps> for Twitter to recognize link properly
+        maps_link_included = True
+        max_length = MAX_TWEET_LEN - TWEET_URL_LEN
+        if ' <gmaps> ' in alert['status']:
+            alert_str = replace(alert['status'].replace(' <gmaps> ', ' '), info)
+        elif '<gmaps> ' in alert['status']:
+            alert_str = replace(alert['status'].replace('<gmaps> ', ''), info)
+        elif ' <gmaps>' in alert['status']:
+            alert_str = replace(alert['status'].replace(' <gmaps>', ''), info)
+        elif '<gmaps>' in alert['status']:                          # Even if string does not have spaces around it, go ahead and process it
+            alert_str = replace(alert['status'].replace('<gmaps>', ''), info)
+        else:                                                       # No gmaps link found
+            maps_link_included = False
+            max_length = MAX_TWEET_LEN
+            alert_str = replace(alert['status'], info)
+            
+        if len(alert_str) > max_length:
+            alert_str2 = "[1/2] " + alert_str[0:(max_length-6)]     # subtract 6 due to the '[1/2] ' string length
```

Comment:
Perhaps it would be better to make this a for loop, incase any users decide they want 3 lines worth of information.